crawler/crawler.conf

```diff
@@ -79,3 +79,4 @@
     access_group_filepath = /etc/sas-secrets/access_group
     cloudoe_filepath = /etc/sas-secrets/cloudoe
     ssl_verification = False
+    emit_interval_filepath = /etc/sas-secrets/emit_interval
```

Comment:
Does it have to be a secret? How about just emit_interval = 10 ?

Reply:
@sahilsuneja1 i think we can keep this parameter as a secret for now..its a temp solution anyway...since, this is one throttling parameter @tatsuhirochiba has added to control rate of frame emission...we expect this to vary based on our experience ..and once we handle this in sas-frontend, we will remove this altogether